src/ui/page/maintenance.html

```diff
@@ -0,0 +1,1179 @@
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-22141655-4">
+    </script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { dataLayer.push(arguments); }
+        gtag('js', new Date());
+        gtag('config', 'UA-22141655-4', { anonymize_ip: true });
+    </script>
+    <style>
```

Comment:
can we move the style configurations (i.e. scss stuff) to a separate file to make this page more readable?

Reply:
it is a copy & paste from error.html page - I can amend